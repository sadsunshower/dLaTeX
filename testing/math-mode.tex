\documentclass{article}

\begin{document}

Test: math mode

I'm using the symbol \theta outside math mode

\end{document}