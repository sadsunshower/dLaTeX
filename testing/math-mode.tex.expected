This is dLaTeX version 0.1
{"messages": [{"severity": "error", "message": "Math mode command / macro used outside of math mode. Did you forget a $ or \\[?", "line": "I'm using the symbol \\theta outside math mode", "num": 7, "marker": [26]}], "error": true}
